app/assets/stylesheets/application.scss

```diff
@@ -27,4 +27,5 @@
 @import 'tables';
 @import 'forms';
 @import 'types';
+@import 'comments';
```

Comment:
Remove this since the imported file is empty. And remove the ` app/assets/stylesheets/comments.scss` file as well.